src/pages/trade/ui/page.tsx

```diff
@@ -1,55 +1,55 @@
 'use client';
 
-import { Card } from '@penumbra-zone/ui/Card';
-import { OrderForm } from './order-form';
-import { PairSelector } from './pair-selector';
-import { RouteBook } from './route-book';
+import cn from 'clsx';
+import { PairInfo } from './pair-info';
 import { Chart } from './chart';
-import { Summary } from './summary';
+import { RouteTabs } from './route-tabs';
+import { TradesTabs } from './trades-tabs';
+import { HistoryTabs } from './history-tabs';
+import { FormTabs } from './form-tabs';
 
 export const TradePage = () => {
   return (
-    <div>
-      <div className='flex gap-2'>
-        <PairSelector />
+    <div
+      className={cn(
+        'grid w-full border-t border-t-other-solidStroke overflow-x-hidden',
+        // mobile grid
+        'grid-cols-[100%] grid-rows-[auto,auto,auto,auto,auto]',
+        // tablet grid (600px)
+        'tablet:grid-cols-[50%,50%] tablet:grid-rows-[auto,auto,auto,auto]',
+        // desktop grid (>900px)
+        'desktop:grid-cols-[1fr,1fr,300px] desktop:grid-rows-[auto,652px,auto,auto]',
+        // large grid (>1200px)
+        'lg:grid-cols-[1fr,320px,320px] lg:grid-rows-[auto,660px,auto]',
+        // extra large grid (>1600px)
+        'xl:grid-cols-[1fr,320px,320px] xl:grid-rows-[auto,660px,auto]',
+      )}
```

Comment:
4 iterations of grid layout implementation led me to these styles. Should be easy enough to extend and add new blocks if needed